cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
-var pipboylib = require('pipboylib')
-var relay = require('pipboylib/lib/relay')
+var pipboylib = require('pipboylib');
+var relay = require('pipboylib/lib/relay');
```

Comment:
I spent so much time removing semicolons :P I use them at work, it will take some time :P 
Before obama there was semistandard (part of the same group of projects as standard)